Tasks/Gradle/sonarqube-common/restresponse.ts

```diff
@@ -0,0 +1,13 @@
+// Data class representing an HTTPS response from the SonarQube server.
+export class RestResponse {
+    constructor(public statusCode:number, public payload:string) {
+    }
+
+    public wasSuccess():boolean {
+        return this.statusCode == 200;
```

Comment:
I am also surprised that there is no such similar call to be made in the default library.